readetexts-port.patch

```diff
@@ -59,17 +59,16 @@ index e8b8022..d7b5d3c 100644
          self.port = 1024 + (h % 64511)
  
          self.is_received_document = False
-@@ -337,7 +338,8 @@ class ReadEtextsActivity(activity.Activity):
+@@ -337,7 +338,7 @@ class ReadEtextsActivity(activity.Activity):
                  self.connect("joined", self.joined_cb)
          elif self.object_id is None:
              # Not joining, not resuming
 -            f = open("help.txt", "r")
-+            script_dir = os.path.dirname(os.path.realpath(__file__))
-+            f = open(os.path.join(script_dir, 'help.txt'), 'r')
++            f = open('help.txt', 'r')
```

Comment:
Note that there are more than one instance of this, just examine the original patches.

Reply:
Also, just keep the original "r", to make this patch smaller.